modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -6,7 +6,7 @@
 [id='ipi-install-firmware-requirements-for-installing-with-virtual-media_{context}']
 = Firmware requirements for installing with virtual media
 
-The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The following table lists supported firmware for installer-provisioned {product-title} clusters deployed with Redfish virtual media.
+The installer for installer-provisioned {product-title} clusters validates the hardware and firmware compatibility with Redfish virtual media. The following table lists the minimum firmware versions tested to work for installer-provisioned {product-title} clusters deployed with Redfish virtual media.
```

Comment:
Given the change in the language here to "minimum", is line 33 necessary? Looking at [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#_v) I don't see a ton of guidance on a `Note` following a `Note`. I do see it recommending trying to reduce multiple admonitions. What I would recommend, perhaps, is omitting line 33 (if it is indeed unnecessary because of your update) and adding a "<1>" to the table section "Dell". This way we can move the longish line 35 to below the table in a corresponding <1> reference. You could, then, combine the `Note` so it was line 25 and line 31. I hope that makes sense. These are, as always, suggestions. Thanks.